mappings/net/minecraft/block/AmethystClusterBlock.mapping

```diff
@@ -7,5 +7,10 @@ CLASS net/minecraft/class_5542 net/minecraft/block/AmethystClusterBlock
 	FIELD field_27091 westShape Lnet/minecraft/class_265;
 	FIELD field_27092 upShape Lnet/minecraft/class_265;
 	FIELD field_27093 downShape Lnet/minecraft/class_265;
+	FIELD field_46255 CODEC Lcom/mojang/serialization/MapCodec;
+	FIELD field_46256 height F
+	FIELD field_46257 xzOffset F
 	METHOD <init> (FFLnet/minecraft/class_4970$class_2251;)V
+		ARG 1 height
+		ARG 2 xzOffset
```

Comment:
What about `horizontalOffset`, or did you name it like that because of the codec field name (`aabb_offset`)?

Reply:
This is the name it had in yarn prior to this snapshot